openApiDocs/beta/Teams.Chats.yml

```diff
@@ -535,15 +542,104 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/microsoft.graph.teamsApp'
-          links:
-            appDefinitions:
-              operationId: teamsAppInstallation.teamsApp.GetAppDefinitions
-              parameters:
-                chat-id: $request.path.chat-id
-                teamsAppInstallation-id: $request.path.teamsAppInstallation-id
-                teamsAppDefinition-id: $response.body#/id
         default:
           $ref: '#/components/responses/error'
+      x-ms-docs-operation-type: operation
+  '/chats/{chat-id}/installedApps/{teamsAppInstallation-id}/teamsApp/$ref':
+    get:
+      tags:
+        - chats.teamsAppInstallation
+      summary: Get ref of teamsApp from chats
+      operationId: chats.installedApps_GetRefTeamsApp
+      parameters:
+        - name: chat-id
+          in: path
+          description: 'key: id of chat'
+          required: true
+          schema:
+            type: string
+          x-ms-docs-key-type: chat
+        - name: teamsAppInstallation-id
+          in: path
+          description: 'key: id of teamsAppInstallation'
+          required: true
+          schema:
+            type: string
+          x-ms-docs-key-type: teamsAppInstallation
+      responses:
+        '200':
+          description: Retrieved navigation property link
+          content:
+            application/json:
+              schema:
+                type: string
+        default:
+          $ref: '#/components/responses/error'
+      x-ms-docs-operation-type: operation
+    patch:
+      tags:
+        - chats.teamsAppInstallation
+      summary: Update the ref of navigation property teamsApp in chats
+      operationId: chats.installedApps_UpdateRefTeamsApp
+      parameters:
+        - name: chat-id
+          in: path
+          description: 'key: id of chat'
+          required: true
+          schema:
+            type: string
+          x-ms-docs-key-type: chat
+        - name: teamsAppInstallation-id
+          in: path
+          description: 'key: id of teamsAppInstallation'
+          required: true
+          schema:
+            type: string
+          x-ms-docs-key-type: teamsAppInstallation
+      requestBody:
+        description: New navigation property ref values
+        content:
+          application/json:
+            schema:
+              type: String
```

Comment:
OpenAPI is case sensitive. $ref request bodies should be of type `string` and not `String`.

Reply:
Shouldn't this type be a dictionary? [OpenAPI dictionary](https://swagger.io/docs/specification/data-models/dictionaries/) i.e..
``` yml
      requestBody:
        description: New navigation property ref values
        content:
          application/json:
            schema:
              type: object
              additionalProperties: true
        required: true
```
cc @darrelmiller, @ddyett